Wrappers/Python/cil/utilities/dataexample.py

```diff
@@ -645,4 +356,150 @@ def scikit_random_noise(image, mode='gaussian', seed=None, clip=True, **kwargs):
+    URL = 'https://zenodo.org/record/4822516/files/walnut.zip'
+    FILE_SIZE = '6.4 GB'
+
+    @classmethod
+    def get(cls, data_dir=DEFAULT_DATA_DIR):
+        '''
+        This function returns the raw projection data from the .txrm file
+
+        Returns
+        -------
+        ImageData
+            The walnut dataset
+        '''
+        self = cls(data_dir)
+        filepath = os.path.join(self.data_dir, cls.__name__, 'valnut','valnut_2014-03-21_643_28','tomo-A','valnut_tomo-A.txrm')
+        try:
+            loader = ZEISSDataReader(file_name=filepath)
+            return loader.read()
+        except FileNotFoundError as exc:
+            raise ValueError(f"Specify a different data_dir or download data with `{cls.__name__}({data_dir}).download_data()`") from exc
+
+class USB(RemoteTestData):
+    '''A microcomputed tomography dataset of a usb memory stick from https://zenodo.org/records/4822516'''
+    URL = 'https://zenodo.org/record/4822516/files/usb.zip'
+    FILE_SIZE = '3.2 GB'
+
+    @classmethod
+    def get(cls, data_dir=DEFAULT_DATA_DIR):
+        '''
+        This function returns the raw projection data from the .txrm file
+
+        Returns
+        -------
+        ImageData
+            The usb dataset
+        '''
+        self = cls(data_dir)
+        filepath = os.path.join(self.data_dir, cls.__name__, 'gruppe 4','gruppe 4_2014-03-20_1404_12','tomo-A','gruppe 4_tomo-A.txrm')
+        try:
+            loader = ZEISSDataReader(file_name=filepath)
+            return loader.read()
+        except FileNotFoundError as exc:
+            raise ValueError(f"Specify a different data_dir or download data with `{cls.__name__}({data_dir}).download_data()`") from exc
+
+class KORN(RemoteTestData):
+    '''A microcomputed tomography dataset of a sunflower seeds in a box from https://zenodo.org/records/6874123'''
+    URL = 'https://zenodo.org/record/6874123/files/korn.zip'
+    FILE_SIZE = '2.9 GB'
+
+    @classmethod
+    def get(cls, data_dir=DEFAULT_DATA_DIR):
+        '''
+        This function returns the raw projection data from the .xtekct file
+
+        Returns
+        -------
+        ImageData
+            The korn dataset
+        '''
+        self = cls(data_dir)
+        filepath = os.path.join(self.data_dir, cls.__name__, 'Korn i kasse','47209 testscan korn01_recon.xtekct')
+        try:
+            loader = NikonDataReader(file_name=filepath)
+            return loader.read()
+        except FileNotFoundError as exc:
+            raise ValueError(f"Specify a different data_dir or download data with `{cls.__name__}({data_dir}).download_data()`") from exc
+
+class SANDSTONE(RemoteTestData):
+    '''
+    A synchrotron x-ray tomography dataset of sandstone from https://zenodo.org/records/4912435
+    A small subset of the data containing selected projections and 4 slices of the reconstruction
+    '''
+    URL = 'https://zenodo.org/records/4912435/files/small.zip'
+    FILE_SIZE = '227 MB'
```

Comment:
btw @hrobarts I think you're missing a `@classmethod def get(...)` here

Reply:
It's not obvious which file you would want to get from the sandstone dataset. I was thinking of just having the download function then the user can load whichever one they need. That's how I would use it in the how-to. But maybe that's a bad idea. What do you think @paskino ?